lib/lightning/invocation.ex

```diff
@@ -393,19 +393,39 @@ defmodule Lightning.Invocation do
         date_after: date_after,
         date_before: date_before
       ) do
-    # we can use a ^custom_query to control (order_by ...) the way preloading is done
+    from(wo in Lightning.WorkOrder,
+      join: wo_re in assoc(wo, :reason),
+      join: w in assoc(wo, :workflow),
+      as: :workflow,
+      join: att in assoc(wo, :attempts),
+      join: r in assoc(att, :runs),
+      as: :runs,
+      join: att_re in assoc(att, :reason),
+      join: d in assoc(r, :input_dataclip),
+      as: :input,
+      where: w.project_id == ^project_id,
+      where: d.id in [wo_re.dataclip_id, att_re.dataclip_id],
+      where: ^filter_workflow_where(workflow_id),
+      where: ^filter_run_status_where(status),
+      where: ^filter_run_started_after_where(date_after),
+      where: ^filter_run_started_before_where(date_before),
+      where: ^filter_run_body_and_logs_where(search_term, searchfors),
+      order_by: [desc_nulls_first: r.finished_at]
+    )
```

Comment:
@wanecode I've split out the 'search' query from the one that preloads the attempts and runs.